### PR TITLE
Material dialog

### PR DESCRIPTION
**Pull request by argelius**

### CODE REVIEW

File: framework/directives/dialog.js [hunk 1]
```diff
@@ -3,6 +3,9 @@
  * @id dialog
  * @name ons-dialog
  * @category dialog
+ * @modifier material
+ *   [en]Display a Material Design dialog.[/en]
+ *   [ja][/ja]
```
Comment (argelius) [inline framework/directives/dialog.js]: @anatoo: Can you please add Japanese documentation here?

**Comment by argelius** [comment]
@anatoo: Please merge this if there is no problems.

**Comment by anatoo** [comment]
@argelius Thanks. I cant find any problems. Merged.

**Comment by argelius** [comment]
Thanks @anatoo!

**Pull request merged**